src/main/scala/scalaz/stream/async/mutable/Queue.scala

```diff
@@ -107,20 +107,31 @@ trait Queue[A] {
   private[stream] def failWithCause(c:Cause): Task[Unit]
 }
 
+private[stream] object CircularBuffer {
+  /**
```

Comment:
Since this is private, it won't show up in generated API docs, so could we just move this description down to the `circularBuffer` method?